tests/integration/cgroups.bats

```diff
@@ -109,8 +109,10 @@ EOF
 }
 
 @test "runc exec (limits + cgrouppath + permission on the cgroup dir) succeeds" {
-   [[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup
-
+   [[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup cgroups_v1
+# FIXME: how to chown/chmod for cgroup2 fs?
+# \"failed to write 86404 to cgroup.procs: write /sys/fs/cgroup/runc-cgroups-integration-test/test-cgroup/cgroup.procs: permissi
```

Comment:
```console
suda@suda-ws01:/sys/fs/cgroup$ sudo mkdir foo
suda@suda-ws01:/sys/fs/cgroup$ sudo chown suda.suda foo
suda@suda-ws01:/sys/fs/cgroup$ mkdir foo/bar
suda@suda-ws01:/sys/fs/cgroup$ echo $$ > foo/bar/cgroup.procs 
-bash: echo: write error: Permission denied
```

cc @giuseppe

Reply:
Looks like a kernel issue (or misfeature) to me. Files created in a new directory should be owned by the same UID/GID as the directory itself. Maybe there's something in v2 docs about it?

Reply:
```console
$ ls -l foo/bar/cgroup.procs 
-rw-r--r-- 1 suda suda 0 Apr 22 07:59 foo/bar/cgroup.procs
```

Owned by me, but EPERM happens

Reply:
https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#delegation-containment

To write `/sys/fs/cgroup/foo/bar/cgroup.procs`, it seems I need to chown/chmod `/sys/fs/cgroup/cgroup.procs` and `/sys/fs/cgroup/cgroup.procs` as well. (The last one is nasty 😕 )

Reply:
I think as a user you're in a non-root cgroup already (`grep ^0: /proc/self/cgroup`) and this is the "source" cgroup to which you need access to, not the root one.